modules/apps/map/map-google-maps/src/main/resources/META-INF/resources/js/GoogleMapsDialog.es.js

```diff
@@ -12,22 +12,18 @@
  * details.
  */
 
-import State, {Config} from 'metal-state';
-
 /**
  * GoogleMapsDialog
  * @review
  */
-class GoogleMapsDialog extends State {
+export default class GoogleMapsDialog {
 
 	/**
 	 * Creates a new map dialog using Google Map's API
 	 * @param  {Array} args List of arguments to be passed to State
 	 * @review
 	 */
-	constructor(...args) {
-		super(...args);
-
+	constructor() {
```

Comment:
We'll need to add a setter/getter for `this.map` here since this component previously extended from `State` and also uses `this.map`

Reply:
Ah, so that's the logic behind when to add the getter and setter. I thought we wanted to add get/set was when there is a `setter` property of that state property.